hladnymatfyzak/__init__.py

```diff
@@ -151,16 +151,17 @@ def ffood(which, day=None, month=None, year=None):
     if month is None:
         month = int(time.strftime("%m"))
     weekday = int((datetime.date(year, month, day)).weekday())
-
+    # weekday should be <0,4>
     req = requests.get(FF_URL)
     soup = BeautifulSoup(req.text)
     menu_week = soup.find_all('ul')
 
     menu_week = menu_week[index]
     menu_bs = BeautifulSoup(str(menu_week))
-    daymenu = menu_bs.find_all('ul')[weekday]
+    # "weekday()" starts at 0, but at site first day starts at 1
```

Comment:
@tWido this looks better (though I still believe it could be improved).

Thanks!